components/crypto/Cargo.toml

```diff
@@ -16,15 +16,17 @@ byteorder = { version = "1.2.7", features = [ "i128" ] }
 chrono = "0.4.6"
 hex = "0.3.2"
 hex-buffer-serde = "0.2.0"
-pwbox = "0.2.0"
+pwbox = { version = "0.2.0", path = "../../../pwbox-rs" }
 rand = "0.7"
 rust_decimal = "1.0"
 serde = "1.0.10"
 serde_derive = "1.0.64"
 serde_json = "1.0.19"
+failure = "0.1.5"
 toml = "0.5.0"
 uuid = "0.7.1"
-exonum_sodiumoxide = { version = "0.0.22", optional = true }
+exonum_sodiumoxide = { git = "http://github.com/pavel-mukhanov/exonum_sodiumoxide/", branch = "key-exchange-mod", optional = true }
+secret-tree = { version = "0.2.0-pre.0", path = "../../../secret-tree/" }
```

Comment:
Is it really a part of the repository ?

Reply:
No, I after release of updated versions of `pwhash`, `secret-tree` and `exonum-sodiumoxide` I will change that.